aiida/orm/querybuilder.py

```diff
@@ -1773,11 +1746,25 @@ def queryhelp(self):
         The queryhelp can be used to create a copy of the QueryBuilder instance like so::
 
             qb = QueryBuilder(limit=3).append(StructureData, project='id').order_by({StructureData:'id'})
-            qb2=QueryBuilder(**qb.queryhelp)
+            qb2 = QueryBuilder(**qb.queryhelp)
 
-        :return: a queryhelp dictionary
+        This is a json-comptible dictionary.
+        In this way, the queryhelp can be stored in the database or a json-object, retrieved or shared and used later.
+        The following is True if no change has been made to the database::
```

Comment:
why not simply append the line `qb.all() == qb2.all()  # true` to the code above?

Reply:
Could've done that, sure. I just preferred the separation of "concepts", but if you prefer them together I can update it?

Reply:
It is done.